src/CMakeLists.txt

```diff
@@ -72,13 +73,31 @@ set(BOT_SOURCES
     WorkerManager.cpp
     WorkerManager.h
     main.cpp
+    ${PROJECT_SOURCE_DIR}/CMakeLists.txt
+    ${PROJECT_SOURCE_DIR}/README.md
 )
 
 add_executable(CommandCenter ${BOT_SOURCES})
 
 # Show more warnings at compiletime.
 if (MSVC)
     target_compile_options(CommandCenter PRIVATE /W4 /EHsc)
+
+    set_target_properties( CommandCenter PROPERTIES
+        VS_DEBUGGER_WORKING_DIRECTORY "$<TARGET_FILE_DIR:CommandCenter>"
+        VS_DEBUGGER_COMMAND "$<TARGET_FILE:CommandCenter>"
+        VS_DEBUGGER_COMMAND_ARGUMENTS "Ladder2019Season3/AcropolisLE.SC2Map"
```

Comment:
Ah, and this point is also confusing as we are in 2023 now and SC2AI ladder uses different maps. While it is ok to specify map from the past as an example of "start the bot" command in README, currently one should use modern maps.

In other words, for real bot development this setting does not make sense and should be changed.
And if we set here some up2date map, we have to update it each maps rotation (i.e. each new ladder season).

Reply:
User can manually update these in the IDE.  These are initially blank variables.  
![FullScreen](https://github.com/cpp-sc2/commandcenter/assets/24380533/91c916aa-833a-4eb6-867b-7894e8059427)
![ProjectProperties](https://github.com/cpp-sc2/commandcenter/assets/24380533/84c7a624-9a3d-45cf-9ee3-0a846ba89369)
It's an initialization, not a permanent change. 
An alternative to launching from the command line.

Reply:
I mean that this alternative is invalid from the very beginning (which is ok for README as it contains examples) so one should figure out which values should be used instead in any case.

I believe all the defaults should contain working values or do not contains values at all. For the latter documentation should be added. The suggested approach does not provide working defaults.